samples/apps/copilot-chat-app/webapp/src/components/chat/prompt-details/PromptDetails.tsx

```diff
@@ -32,8 +32,8 @@ export const PromptDetails: React.FC<IPromptDetailsProps> = ({ message }) => {
                 <DialogBody>
                     <DialogTitle>Prompt</DialogTitle>
                     <DialogContent>
-                        {(message.prompt === undefined || message.prompt === '')
-                            ? 'Empty'
+                        {(message.prompt === undefined || message.prompt === '' || message.prompt === null)
+                            ? 'No prompt available. The response is either a plan proposal or an initial greeting message.'
```

Comment:
```suggestion
                            ? 'No prompt available. The response is either a plan proposal or auto-generated response.'
```

To encompass responses like "No, cancel" on plan rejection

Reply:
I would change it to "hard-coded response". "auto-generated response" sounds like what we are already doing with the AI, lol